ui/lib/pki/addon/routes/issuers/index.js

```diff
@@ -0,0 +1,24 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class PkiIssuersIndexRoute extends Route {
+  @service store;
+  @service secretMountPath;
+  @service pathHelp;
+
+  async model() {
+    // the pathHelp service is needed for adding openAPI to the model
+    await this.pathHelp.getNewModel('pki/pki-issuer-engine', 'pki');
+
+    let response = await this.store
+      .query('pki/pki-issuer-engine', { backend: this.secretMountPath.currentPath })
+      .catch((err) => {
+        if (err.httpStatus === 404) {
+          return [];
+        } else {
+          throw err;
+        }
+      });
+    return response;
```

Comment:
Would it be better to return the result of the query directly rather than assigning it to the `response` variable?

Reply:
good point. Yep. I'll amend.